docs/api/public_php_api_search.md

```diff
@@ -244,6 +248,14 @@ You can access the following additional search result data from PagerFanta:
 |[`LocationSearchHitAdapter`](https://github.com/ibexa/core/blob/main/src/lib/Pagination/Pagerfanta/LocationSearchHitAdapter.php)|Makes a Location search against passed Query and  returns [SearchHit](https://github.com/ibexa/core/blob/main/src/contracts/Repository/Values/Content/Search/SearchHit.php) objects instead.|
 |[`ContentFilteringAdapter`](https://github.com/ibexa/core/blob/main/src/lib/Pagination/Pagerfanta/ContentFilteringAdapter.php)|Applies a Content filter and returns a `ContentList` object.|
 |[`LocationFilteringAdapter`](https://github.com/ibexa/core/blob/main/src/lib/Pagination/Pagerfanta/LocationFilteringAdapter.php)|Applies a Location filter and returns a `LocationList` object.|
+|`AttributeDefinitionListAdapter`| Makes a search for product attributes and returns an `AttributeDefinitionListInterface` object. |
+|`AttributeGroupListAdapter`| Makes a search for product attribute groups and returns an `AttributeGroupListInterface` object. |
+|`CurrencyListAdapter`| Makes a search for currencies and returns a list of currencies. |
```

Comment:
`\Ibexa\Contracts\ProductCatalog\Values\Currency\CurrencyListInterface` is returned here. Therefore:

```suggestion
|`CurrencyListAdapter`| Makes a search for currencies and returns a `CurrencyListInterface` object.
```

Although it has to be said, that the fact that a specific object type is returned is by no means guaranteed - for Pagerfanta pagination all that is important really is that it will be an iterable list of certain type of objects.

The fact that it is represented by such list object is an implementation detail.

Reply:
The whole table lists returned objects, so that should probably be removed for all, outside this PR